week-3/Homework/mandatory/3-space-colonies.js

```diff
@@ -13,7 +13,21 @@
   
 */
 
-function colonisers() {}
+function colonisers(listOfNames) {
+
+  function makeSubArr(listOfNames){
+    return listOfNames.includes('family');
+  }
+
+  let familyArr = listOfNames.filter(makeSubArr);
+
+  function findFamiliesA(subArr){
+    return subArr.includes('A');
```

Comment:
Can you think of what would happen if you had a family name with a `"A"` in the middle of the string - it might not work in this case. However, you can also think about using `startsWith()` too.